src/main/java/edu/cornell/kfs/sys/dataaccess/impl/DocumentMaintenanceDaoJdbc.java

```diff
@@ -38,22 +38,23 @@
 
     private CuSqlQuery buildRequeueSqlQuery() {
         CuSqlChunk sqlChunk = buildRequeueSqlQueryChunk();
-        sqlChunk.append(" ORDER BY DOC_HDR_ID ASC");
+        sqlChunk.append(" ORDER BY DH.DOC_HDR_ID ASC");

```

Comment:
All of the other methods that have a table alias defined keep that alias in the scope of the method. Even though this method is private to this class, I am concerned about the table alias being outside of the method where is is defined.  
What are your thoughts about the use of a boolean parameter "includeOrderBy" to the buildRequeueSqlQueryChunk method to keep this SQL within the method scope of the table alias definition which would also allow the buildRequeueSqlQueryChunk to be reusable?

Reply:
Ok, I'll see if I can adjust this to get it closer to where the alias was defined. I was hoping to avoid the use of a boolean method parameter, but now that an alias has been added for that table, I agree it makes more sense to move it over to where the alias was defined.